.prettierrc

```diff
@@ -1,7 +1,8 @@
 {
+  "arrowParens": "avoid",
+  "printWidth": 80,
   "singleQuote": true,
   "semi": false,
   "trailingComma": "all",
-  "tabWidth": 2,
-  "printWidth": 80
+  "tabWidth": 2
```

Comment:
This was part of the motivation for including prettier. By default, tsc transpilation introduces 4-space indentation, which would have been disruptive and made it harder to identify changes to the JS snippets.